src/main/java/org/spongepowered/api/item/recipe/Recipe.java

```diff
@@ -47,29 +44,10 @@
 public interface Recipe {
 
     /**
-     * Returns the list of item types that result when successful crafting of
-     * this Recipe is completed.
+     * Gets the result of this {@link Recipe}.
      *
-     * @return The resultant list of item types
+     * @return the results of this {@link Recipe}
      */
-    List<ItemType> getResultTypes();
-
-    /**
-     * Checks if the given {@link GridInventory} fits the required constraints
-     * to craft this Recipe.
-     *
-     * @param grid The ItemGrid to check for validity
-     * @return True if the given input matches this recipe's requirements
-     */
-    boolean isValid(GridInventory grid);
```

Comment:
Should be left in as deprecated, potentially default implemented to return true? I'm not sure about it, but try to have as minimal of a breakage as possible.